Luchynska Tetiana/my-todo-list/src/app/app.component.html

```diff
@@ -0,0 +1 @@
+<app-todo></app-todo>
```

Comment:
It'll be better to name application - 'app-todo-list'

Reply:
ok, changed